public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```

Comment:
Our name?

Reply:
We could start with "kubevirt web ui" and the long would be like "A UI for Managing Kubevirt".
What you think?

Reply:
Kubevirt? :-D 
If so, I would rename whole project to "kui" or better "kua" (if anyone asks then Kubevirt UserInterface in Action). The second one sounds best to me.

If Kubevirt is not an option in the name, then what about "Virtualization User Interface for Kubernetes"

Reply:
Naming things... I would not care about this for now, I suggest open a specific PR (after this one) to discuss the name if you like, but not block getting the foundations in.

Reply:
> manifest.json provides metadata used when your web app is added to the homescreen on Android

So I guess we can remove this file :smile: 

As for the naming, it's currently `kubevirt-web-ui` in `package.json` file, and I'm all for a better name!